Dockerfile

```diff
@@ -109,11 +109,12 @@ WORKDIR /work/
 # hadolint ignore=DL3041
 RUN microdnf update --refresh --best --nodocs --noplugins --setopt=install_weak_deps=0 -y \
     && microdnf install ca-certificates libcap shadow-utils systemd --best --nodocs --noplugins --setopt=install_weak_deps=0 -y \
-    && microdnf clean all \
-    && rm -rf /var/cache/yum \
     && chmod -R 777 . \
     && setcap "cap_dac_read_search+p" /work/logdna-agent \
     && groupadd -g 5000 logdna \
-    && useradd -u 5000 -g logdna logdna
+    && useradd -u 5000 -g logdna logdna \
+    && microdnf remove shadow-utils -y \
+    && microdnf clean all \
+    && rm -rf /var/cache/yum
```

Comment:
`shadow-utils` isn't required after `useadd` and `groupadd`, so save a little space here.